src/view/ChewingEditor.cpp

```diff
@@ -104,6 +104,8 @@ void ChewingEditor::execFileDialog(DialogType type)
 
 void ChewingEditor::selectImportFile()
 {
+    import_count = 0;
```

Comment:
Correct me if I'm wrong, _"Then, you don't have to expose it in the class declaration."_ means that I should remove `import_count` from the class `ChewingEditor` as they are independent.
In replacement, I should directly declare a static variable `import_count` in the file "ChewingEditor.cpp" and manipulate with it within "ChewingEditor.cpp".
Is it OK if I declare it as a global variable as I would need to reset its value back to 0 whenever `ChewingEditor::selectImportFile()` is invoked.